awx/main/conf.py

```diff
@@ -775,6 +775,23 @@
     help_text=_('Indicates whether the instance is part of a kubernetes-based deployment.'),
 )
 
+register(
+    'SUBSCRIPTION_USAGE_MODEL',
+    field_class=fields.ChoiceField,
+    choices=[
+        ('', _('Default model for AWX - no subscription')),
```

Comment:
I want to leave this as a non-blocking comment, but still want it to be a very strong suggestion.

To improve the coherency of docs, I would add a sentence to the end of this - "Deletion of host_metrics will not be considered for purposes of managed host counting." Otherwise, I feel like the numerical consequence of the option is lost, or left too vague to understand.

Reply:
https://github.com/ansible/awx/pull/13736